### DISCUSSION TITLE
cylindric custom support

### DISCUSSION THREAD

**Discussion by petelton** [community/community#133764] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi. I am using Ultimaker Cura 5.6 release, however the plugin "cylindric custom support" is not available from marketplace to install.
Any one are facing this issue?
How can I fix it this?
Thanks

**Comment by akibhaider** [comment]
You can try to find it from GitHub or the plugin developer's website. Some developers host their plugins outside the Cura Marketplace:
GitHub: Check GitHub repositories related to the plugin. You can often download the plugin files and manually add them to Cura.
Developer's Website or Forums: If you know any relevant developer or community, visit their website or forums for information on obtaining or installing the plugin manually.

**Comment by AnasCodex** [comment]


**Reply by ebndev** [reply]
Hi @AnasCodex, thanks for participating in GitHub Discussions!

Please note: we’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Please feel encouraged to continue engaging in Discussions as your authentic self though. We appreciate you being a part of the Community 🌟

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
